biojava-structure/src/main/java/org/biojava/nbio/structure/StructureTools.java

```diff
@@ -2019,7 +2019,9 @@ public static void cleanUpAltLocs(Structure structure) {
 	 * Expands the NCS operators in the given Structure adding new chains as needed.
 	 * The new chains are assigned ids of the form: original_chain_id+ncs_operator_index+"n"
 	 * @param structure
+	 * @deprecated use {@link org.biojava.nbio.structure.xtal.CrystalBuilder#expandNcsOps(Structure, Map, Map)} instead.
 	 */
+	@Deprecated
```

Comment:
There's no need to deprecate this one. It was an addition for 5.0, so it wasn't ever public yet. I'd say let's remove it.